rust/fastsim-core/src/simdrive/cyc_mods.rs

```diff
@@ -332,8 +332,9 @@ impl RustSimDrive {
             };
             let g = self.props.a_grav_mps2;
             let m = self.veh.veh_kg;
-            let rho_cdfa =
-                self.props.air_density_kg_per_m3 * self.veh.drag_coef * self.veh.frontal_area_m2;
+            let rho_cdfa = self.veh.drag_coef.clone()
```

Comment:
@kylecarow , `VehiclField<f64>` can be multiplied by f64 but not the other order.  Because types that are used in math operations normally implement the `std::markers::Copy` trait and `VehicleField<f64>` cannot implement that trait due to fields that can't do it, we need to use `.clone()` a lot, which in my mind negates the benefit of the `std::ops` implementations.